src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const set = new Set(values);
+
+  if (values.length !== set.size) {
+    return null;
+  }
+
+  const invRobot = {};
+
+  Object.keys(robot).forEach(key => {
+    const newKey = robot[key];
+
+    invRobot[newKey] = key;
+  });
```

Comment:
Here you can just check if the robot already has the same own property and if it does - return 'null', 
in other case - `invRobot[newKey] = key;` as you've already wrote.

And it's way more simple that use Set and Object.values and compare their sizes.